docs/source/config.rst

```diff
@@ -11,47 +11,47 @@ File format
 **********************************
 
 
-The top level structure is a dict with one ``"systems"`` key with a dict value, containing
+The top level structure is a single dict with one ``"systems"`` key with a dict value, containing
 one key-value pair for each system. The key is the name of the system (used to select
 the system when evaluating the function).  See ``expyre.System`` docstring for full list. The value is a dict with the following keys:
 
 - ``"host"``: string hostname, with optional ``username@`, or ``null`` for localhost without ssh.
 - ``"scheduler"``: string indicating type of scheduler, currently ``"slurm"``, ``"pbs"`` or ``"sge"``
 - ``"commands"``: optional list(str) with commands to run at beginning of every script on system, usually for things that set up the runtime such as environment modules (e.g. ``module load vasp``)
-- ``"header"``: list(str) queuing system header lines.  Actual header is created by applying string formatting, i.e.  ``str.format(**kwargs)``, replacing substrings such as ``{nnodes}``.
+- ``"header"``: list(str) queuing system header lines.  Actual header is created by applying string formatting, i.e.  ``str.format(**node_dict)``, replacing substrings such as ``{nnodes}``.
```

Comment:
I think that's `num_nodes` now?

Reply:
I guess not, because `nnodes` is used internally in the code. Would you mind straightforwardly replacing `nnodes` -> `num_nodes` & `ncores` -> `num_cores` across the code, so there's no confusion?

Reply:
I think I got it everywhere now.  I updated the creation of `node_dict`, and also where it's used in the github CI config file and the tests.